params/computation_cost_params.go

```diff
@@ -171,6 +169,14 @@ const (
 	XorComputationCost            = 657
 	XorComputationCostIstanbul    = 454
 
-	// computation costs for opcode added at koreCompatible Protocol Upgrade
+	// computation cost added at KoreCompatible
 	RandomComputationCost = 1498
+
+	// computation cost added at ShanghaiCompatible
+	Push0ComputationCost = 80
+
+	// computation cost added at CancunCompatible
+	McopyComputationCost  = 250
+	TloadComputationCost  = 300
+	TstoreComputationCost = 540
```

Comment:
Load and Store implementations seem not much different. The additional work of journal appending seems not add too much to runtime cost, maybe addition 10-20 would be reasonable. By the way, what insight brings you to acquire these numbers?

Reply:
`Load` calls stack.peek() or getTransient functions which references an element of stack
`Store` calls stack.pop() two times. I found that stack.pop() gives a time difference than I thought.

I have run benchmark tests in `instructions_test.go` to get the computation cost value as we did at https://github.com/klaytn/klaytn/pull/1986.

However, I ran the test again and found that this value is pretty big.. let me modify those values to 220, 280.

Reply:
Applied at https://github.com/klaytn/klaytn/pull/1988/commits/9fe8b33956ae04106cf25a226c1533ef032d0627

Reply:
Applied at https://github.com/klaytn/klaytn/pull/1988/commits/87f41c4ee847d5f9605aca079e10244b67b68a5f